index.html

```diff
@@ -3,27 +3,26 @@
   <head>
     <meta charset="utf-8"/>
     <title>Web Chat: Markdown</title>
-    
+
     <script src="https://cdn.botframework.com/botframework-webchat/master/webchat.js"></script>
-    <script src="https://cdn.botframework.com/botframework-webchat/4.2.1-master.14eebbb/webchat-es5.js"></script>
+    <script src="https://unpkg.com/core-js@2.6.3/client/core.min.js"></script>
+    <script src="https://unpkg.com/whatwg-fetch@3.0.0/dist/fetch.umd.js"></script>

```

Comment:
`whatwg-fetch` give you `fetch`, but it requires `Promise` (which is provided by `core-js` the line above).